src/site/_DevSiteApplicationContainer.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { useRouteMatch } from 'react-router-dom';
+import PropTypes from 'prop-types';
+import ApplicationContainer from '@cerner/terra-application/lib/application-container/ApplicationContainer';
+
+import { titleConfigShape } from './siteConfigShapes';
+
+const propTypes = {
+  /**
+   * The A session provider
+   */
+  children: PropTypes.node,
+  /**
+   * The title config for the app, used for the container's application name.
+   */
+  titleConfig: titleConfigShape.isRequired,
+};
+
+const DevSiteApplicationContainer = ({ titleConfig, children }) => {
+  const isRaw = useRouteMatch('/raw');
+  return (
+    <ApplicationContainer applicationName={titleConfig.titleConfig} skipToLinksAreDisabled={!!isRaw} unloadPromptIsDisabled={!!isRaw}>
```

Comment:
??

```suggestion
    <ApplicationContainer applicationName={titleConfig.title} skipToLinksAreDisabled={!!isRaw} unloadPromptIsDisabled={!!isRaw}>
```

Reply:
that makes sense, you probably don't want `undefined` to be your application name.

Reply:
updated